6.0/fpm/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM php:5.6-fpm
+
+RUN apt-get update && apt-get install -y \
+  bzip2 \
+  libcurl4-openssl-dev \
+  libfreetype6-dev \
+  libicu-dev \
+  libjpeg-dev \
+  libmcrypt-dev \
+  libmemcached-dev \
+  libpng12-dev \
+  libpq-dev \
+  libxml2-dev \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
Swap the spaces to tabs lines 4-14. Applies to 6.0, 7.0, 8.0, and 8.1 Dockerfiles.